desktop.css

```diff
@@ -350,7 +350,7 @@ h5 {
 }
 
```

Comment:
- [ ] [OPTIONAL] In your `about-me` class I it would be great if you add these 2 lines to fix your project because there is some additional part in background that you can see from this screenshot 🔍👇

To add inside `about-me` class
```css
padding-bottom: 50px;
margin-bottom: -50px;
```
Screenshot

![image](https://user-images.githubusercontent.com/69011963/144710952-665692d4-c41d-4217-8ed0-316ae822028b.png)